apps/website/src/components/EditPageLink.astro

```diff
@@ -0,0 +1,33 @@
+---
+import EditIcon from '~/icons/EditIcon.astro';
+
+type Props = { page: string };
+const { page } = Astro.props;
+
+const githubEditUrl = `https://github.com/iTwin/iTwinUI-react/edit/main/apps/website/src/pages/${page}.mdx`;
```

Comment:
Future problem: I think it's possible to add this link to config to not have hardcoded values. When we deploy this page to cloud, we could just update config and not full app if we ever change docs location.

Reply:
Actually same with all links.

Reply:
I'm not sure I understand how the place we *deploy* affects this link. The github link will stay the same regardless of whether it is deployed to a custom domain or one from azure or netlify

Reply:
True. I just thought it would be better practise to have customisable settings in config file. But probably we are not going to change that often or in runtime (eg. other apps have different URLs for dev, qa and prod environments).